_includes/hero.html

```diff
@@ -5,10 +5,11 @@
 <section class="usa-hero">
   <div class="grid-container">
     <div class="usa-hero__callout">
-      <h1 class="usa-hero__heading"><span class="usa-hero__heading--alt">Hero callout:</span>Bring attention to a project priority
+      <h1 class="usa-hero__heading">This is not the Guides home page
       </h1>
-      <p>Support the callout with some short explanatory text. You don’t need more than a couple of sentences.</p>
-      <a class="usa-button" href="javascript:void(0)">Call to action</a>
+      <p>On the web, this home page does not appear. Look into the individual
+      guides below for the actual Guides content.</p>
+      <a class="usa-button" href="javascript:void(0)">Look elsewhere</a>
```

Comment:
```suggestion
      <a class="usa-button" href="https://18f.gsa.gov/guides/">Leave to visit the actual Guides homepage</a>
```

Reply:
Maybe link to the actual homepage 🤷🏾 
We might also consider just removing this link.